package.json

```diff
@@ -20,10 +20,12 @@
     "url": "https://github.com/civicsource/knockout.typeahead/issues"
   },
   "homepage": "https://github.com/civicsource/knockout.typeahead#readme",
-  "peerDependencies": {
-    "knockout": "3.x",
+  "dependencies": {
     "jquery": "x",
```

Comment:
does the single x work? why not just do _>=1.10_ or whatever version it is that we use